plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx

```diff
@@ -31,10 +31,21 @@ export const SettingsPaymentsMain = () => {
 	);
 	const { installAndActivatePlugins } = useDispatch( PLUGINS_STORE_NAME );
 
+	// Get WooCommerce store country, split by colon to avoid the state.
+	const storeDefaultCountry =
+		window.wcSettings?.admin?.preloadSettings?.general?.woocommerce_default_country?.split(
```

Comment:
I guess we could improve it here (probably) since if split fails, then we might now have `[0]` item

Reply:
I don't think we need any of this. The fallback should happen on the backend.

Reply:
I agree with Vlad. We can use `WC()->countries->get_base_country()` on the backend where retrieve the country from the meta.

Reply:
The problem is that the current provider route doesn't send the country back, only a list of providers/suggestions/etc...

I would also vote to add this to the returned schema and use it instead of relying on the `wcSettings` here

Reply:
I think these [two](https://github.com/woocommerce/woocommerce/pull/53505/commits/1040433148e0d570d08582bb043cff9e015b3c2a) [commits](https://github.com/woocommerce/woocommerce/pull/53505/commits/e90249c7d3f000d7eea0bf26458ff4342102dea0) solve our conundrum :)